src/vivarium_public_health/risks/effect.py

```diff
@@ -112,8 +112,13 @@ def __init__(self, risk: str, target: str):
     def setup(self, builder: Builder) -> None:
         self.exposure = self.get_risk_exposure(builder)
 
-        self.target_modifier = self.get_target_modifier(builder)
-
+        self.relative_risk = self.get_relative_risk(builder)
+        self.relative_risk_pipeline = builder.value.register_value_producer(
```

Comment:
You should extract this call to a method called `get_relative_risk()`. Then you don't need to rename the method by that name in `LBWSGRiskEffect`.